lib/api.js

```diff
@@ -757,9 +798,14 @@ API.prototype.signTxProposalFromAirGapped = function(txp, encryptedPkr, m, n) {
   self.credentials.n = n;
   self.credentials.addPublicKeyRing(publicKeyRing);
 
-  if (!Verifier.checkTxProposal(self.credentials, txp)) {
-    throw new Error('Fake transaction proposal');
-  }
+  // When forAirGapped=true -> checkTxProposal is sync
```

Comment:
Making the `signTxProposalFromAirGapped` method async is the right thing to do IMO.